downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md

```diff
@@ -93,6 +93,21 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ### New features
 
+#### Support for regular expressions as a first-class element
```

Comment:
Or
```suggestion
#### Language support for regular expressions
```

?

Reply:
`Language support for regular expressions`, this is already there with the help of stdlibs?
So don't we need to say we are adding regex as a first class element

Reply:
That's part of the stdlibs. Since we have a separate Language vs stdlib separation in the release note also, IMO, it is OK.

Reply:
Should be fine. +1